### DISCUSSION TITLE
Welcome to ShoppingWebsite Discussions!

### DISCUSSION THREAD

**Discussion by SukeshReddyR** [SukeshReddyR/ShoppingWebsite#3] [Announcements]
### Discussed in https://github.com/SukeshReddyR/ShoppingWebsite/discussions/1

<div type='discussions-op-text'>

<sup>Originally posted by **SukeshReddyR** September 27, 2022</sup>

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

</div>

**Comment by SukeshRondla** [comment]
I am in

**Reply by SukeshReddyR** [reply]
Thank you

**Comment by michelleharris6173** [comment]
in
